ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -3365,6 +3365,55 @@ def test_torch_instance_argmin(
     )
 
 
+# argsort
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="torch.tensor",
+    method_name="argsort",
+    dtype_value=helpers.dtype_and_values(
```

Comment:
You can use the same helper function to generate both arrays and axis `helpers.dtype_value_axis` to avoid unnecessary data generation. 
And as `complex` dtypes are not supported by `argsort` you will change `valid` to `numeric`

Reply:
thanks  for the suggestion, I'll update and mark it.
ps `helpers.dtype_value_axis` didn't return any result in docs, apparently it `helpers.dtype_values_axis` missing a s in 'value'. Although I am surprised how neither sphinx nor github search return any meaningful results and I had grep it locally.